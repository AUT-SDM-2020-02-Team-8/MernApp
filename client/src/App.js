import React, { Component } from "react"
import { BrowserRouter as Router, Route } from "react-router-dom"

import { Provider } from "react-redux"
import store from "./store"
import { Container, AppBar, Typography, Toolbar } from "@material-ui/core"
import { styled } from '@material-ui/core/styles'
import SearchContainer from './components/search/SearchContainer'

const BodyContainer = styled(Container)({
  marginTop: '30px'
})

class App extends Component {
  render() {
    return (
      <Provider store={store}>
        <Router>
          <AppBar position='static'>
            <Container>
              <Toolbar>
                <Typography variant="h6">
                  SEER APP TEAM 8 - SDM 2020
                </Typography>
              </Toolbar>
            </Container>
          </AppBar>
          <BodyContainer>
            <Route exact path="/" component={SearchContainer} />
          </BodyContainer>
        </Router>
      </Provider>
    );
  }
}
export default App;
