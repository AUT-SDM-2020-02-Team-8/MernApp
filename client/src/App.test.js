import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import App from './App';

test ('reders homepage',() =>{
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
}
)
