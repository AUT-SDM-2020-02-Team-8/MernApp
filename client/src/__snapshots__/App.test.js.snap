// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <WithStyles(ForwardRef(AppBar))
      position="static"
    >
      <WithStyles(ForwardRef(Container))>
        <WithStyles(ForwardRef(Toolbar))>
          <WithStyles(ForwardRef(Typography))
            variant="h6"
          >
            SEER APP TEAM 8 - SDM 2020
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(Toolbar))>
      </WithStyles(ForwardRef(Container))>
    </WithStyles(ForwardRef(AppBar))>
    <Styled(WithStyles(ForwardRef(Container)))>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/create"
      />
    </Styled(WithStyles(ForwardRef(Container)))>
  </BrowserRouter>
</Provider>
`;
